app/views/organizations/list_requests.html.erb

```diff
@@ -21,7 +21,7 @@
       <td><%= organization.created_at %></td>
       <td><%= link_to 'Show information', organization %></td>
       <td><%= link_to 'Accept', accept_organization_path(organization), method: :post, class: 'btn btn-success' %></td>
-      <td><%= link_to 'Reject', reject_reason_input_organization_path(organization), method: :post, class: 'btn btn-danger' %></td>
+      <td><%= link_to 'Reject', reject_reason_input_organization_path(organization), class: 'btn btn-danger' %></td>
```

Comment:
The 'Reject' link does not actually reject the organization. It redirects to this page https://github.com/testmycode/tmc-server/blob/tmc-teacher-tools/app/views/organizations/reject_reason_input.html.erb where admin can give a reason for the rejection and then reject with a post request.